staging/src/k8s.io/dynamic-resource-allocation/structured/allocator.go

```diff
@@ -193,3 +193,8 @@ var availableAllocators = []struct {
 		},
 	},
 }
+
+// NodeMatches wraps stable.NodeMatches from internal/stable to make it available to other packages.
```

Comment:
Hmm, this assumes that the implementation is the same across all implementations. What if features are active where e.g. `experimental.NodeMatches` behaves differently? See the comment about `PerDeviceNodeSelection`...